### PR TITLE
LRAC-15111

### PR DESCRIPTION
**Pull request by ealonso**
On Hold: Waiting for AC team confirmation

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/servlet/taglib/renderer/LayoutStructureRenderer.java [hunk 1]
```diff
@@ -210,7 +212,22 @@ private void _renderCollectionStyledLayoutStructureItem(
...
-		jspWriter.write("<div class=\"");
+		jspWriter.write("<div");
+
+		if (FeatureFlagManagerUtil.isEnabled("LRAC-14922")) {
+			ListObjectReference listObjectReference =
+				renderCollectionLayoutStructureItemDisplayContext.
+					getListObjectReference();
+
+			if (listObjectReference != null) {
+				jspWriter.write(" analytics-targetable-collection=\"");
```
Comment (interaminense) [inline modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/servlet/taglib/renderer/LayoutStructureRenderer.java:223]: Hi @ealonso, can you rename it?
```suggestion
				jspWriter.write(" data-analytics-targetable-collection=\"");
```

This way we will be able to get data from js like: `foo.dataset.analyticsTargetableCollection`

File: modules/apps/layout/layout-api/src/main/java/com/liferay/layout/list/retriever/KeyListObjectReference.java [hunk 2]
```diff
@@ -31,6 +32,12 @@ public String getTitle() {
 		return _title;
 	}
 
+	@Override
+	public String toString() {
+		return StringBundler.concat(
+			"{key=", _key, ", itemType=", _itemType, ", title=", _title, "}");
```
Comment (interaminense) [inline modules/apps/layout/layout-api/src/main/java/com/liferay/layout/list/retriever/KeyListObjectReference.java:38]: Is it possible to rename from `=` to `:`? This way we will be able to transform data in a object on frontend.

**Review by interaminense — COMMENTED** [review]
Hi @ealonso, I left some comments on your PR. Thanks a lot for working on it.

File: modules/apps/layout/layout-api/src/main/java/com/liferay/layout/list/retriever/KeyListObjectReference.java (hunk 2 above)
Comment (ealonso) [reply modules/apps/layout/layout-api/src/main/java/com/liferay/layout/list/retriever/KeyListObjectReference.java:38]: sure

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**